### ISSUE TITLE
[Feature Request] Immediate capture when mouse button released after rectangle selection

### ISSUE THREAD

**Issue by liljom** [pop-os/cosmic-screenshot#61]

**Comment by LinuxBoy-96** [comment]
I disagree, sometime you started at the wrong place and will need to make some ajustemnts.

**Comment by leviport** [comment]
It would most likely be a non-default setting, if implemented.

**Comment by KodeBarista** [comment]
Perhaps a better way to implement this would be to allow the middle mouse button to capture the screenshot. This would let users capture it as soon as the selection is complete, without having to press the <kbd>Enter</kbd> key on the keyboard, which is just as fast anyway.

**Comment by NoamVH** [comment]
If I may add - I have one of the buttons on my mouse set to be Ctrl+C. So I just press Super + Shift + S to open Cosmic Screenshot, choose the area I need, and press that button to copy the selection to the clipboard, so it's a comfortable way to get a quick screenshot.
